docker-compose.yml

```diff
@@ -35,7 +39,18 @@ services:
       - 9100
     networks:
       - back-tier
-  
+  alertmanager:
+    image: prom/alertmanager
+    ports:
+      - 9093:9093
```

Comment:
does alertmanager really need open ports? isnt `expose` enough so service prometheus can reach it internally?